h/api/test/schemas_test.py

```diff
@@ -237,18 +237,8 @@ def test_it_raises_if_group_is_not_a_string(self):
 
         assert str(err.value) == "group: False is not of type 'string'"
 
-    def test_it_raises_if_permissions_is_missing(self):
```

Comment:
[It's no longer required](https://github.com/hypothesis/h/pull/3256/commits/34b8c2a6a5ae7d36850fc65b0582e11c4a7de2a1) because:

* When creating it defaults to shared (which I'm planning to change to not shared)
* When updating it defaults to no change
* Also to make permissions consistent with the rest of the fields (it was the only required one)